ansible-playbook.yml

```diff
@@ -0,0 +1,67 @@
+- name: Cyberspark13 Ansible
+  hosts: nodes
+  remote_user: ec2-user
+  vars_prompt:
+    - name: ansible_trello_api_key
+      prompt: What is your Trello API Key?
+      private: yes
+    - name: ansible_trello_api_secret_key
+      prompt: What is your Trello API Secret?
+      private: yes
+    - name: ansible_trello_board_key
+      prompt: What is your Trello Board ID?
+      private: yes
+
+  tasks:
+  - name: Install latest Git and Python3
+    yum:
+      name:
+        - git
+        - python3
+      state: latest
+    become: yes
+
+
+  - name: Install the latest version of poetry via curl and python
+    ansible.builtin.shell:
+      cmd: curl -sSL https://install.python-poetry.org | python3 -
+    args:
+      creates: ~/.local/bin/poetry
+
+  - name: Create App directory
+    ansible.builtin.file:
+      path: /opt/todoapp
+      state: directory
+      owner: ec2-user
+    become: yes
+
+  - name: App git checkout
+    ansible.builtin.git:
+      repo: 'https://github.com/cyberspark13/DevOps-Course-Starter.git'
+      dest: /opt/todoapp
+      version: module_4
+      force: yes
```

Comment:
Got caught out making some manual modifications? I think this is a reasonable default, the docs say it used to be (and originally it wasn't even configurable to turn it off)